### PR TITLE
feat(Page): add page section main nav type variant

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Adding missing page main nav section

**Refer to issue**: #1742

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Page/PageSection.tsx [hunk 1]
```diff
@@ -7,15 +7,22 @@ export enum PageSectionVariants {
...
+}
 
 export interface PageSectionProps extends React.HTMLProps<HTMLDivElement> {
   /** Content rendered inside the section */
   children?: React.ReactNode;
   /** Additional classes added to the section */
   className?: string;
   /** Section background color variant */
-  variant?: 'default' | 'light' | 'dark' | 'darker' | PageSectionVariants
+  variant?: 'default' | 'light' | 'dark' | 'darker' | PageSectionVariants;
```
[the remark this answers is not in the document]
Comment (kmcfaul) [reply packages/patternfly-4/react-core/src/components/Page/PageSection.tsx]: These came in from the TS conversion, which I followed for the type prop I added. Should I remove the enums and just leave the unions?

File: packages/patternfly-4/react-core/src/components/Page/PageSection.tsx [hunk 2]
```diff
@@ -7,15 +7,22 @@ export enum PageSectionVariants {
...
 
 export interface PageSectionProps extends React.HTMLProps<HTMLDivElement> {
   /** Content rendered inside the section */
   children?: React.ReactNode;
   /** Additional classes added to the section */
   className?: string;
   /** Section background color variant */
-  variant?: 'default' | 'light' | 'dark' | 'darker' | PageSectionVariants
+  variant?: 'default' | 'light' | 'dark' | 'darker';
+  /** Section type variant */
```
Comment (jschuler) [inline packages/patternfly-4/react-core/src/components/Page/PageSection.tsx]: when should consumers use nav? should there be an example? maybe a little more descriptive comment?

**Comment by srambach** [comment]
Results look good to me 👍

File: packages/patternfly-4/react-core/src/components/Page/PageSection.tsx (hunk 2 above)
Comment (tlabaj) [reply packages/patternfly-4/react-core/src/components/Page/PageSection.tsx]: Agreed. There is no example in core.  @srambach  will be opening up an additional issue to add one.

**Pull request merged**